sessiontxn/staleread/processor.go

```diff
@@ -237,7 +259,7 @@ func (p *staleReadProcessor) evaluateFromStmtTSOrSysVariable(stmtTS uint64) erro
 	}
 
 	// Otherwise, it means we should not use stale read.
-	return p.setAsNonStaleRead()
+	return p.evaluateFromExternalTS()
```

Comment:
The comment and the method call is not match. Can we handle the extenal ts like others? for example:

```
if ts := p. evaluateFromExternalTS(); ts > 0 {
    return p.setEvaluatedTS(externalTimestamp)
}

// Otherwise, it means we should not use stale read.
return p.setAsNonStaleRead()
```

Reply:
Fixed. The only difference is that I added as new function to set ts but not set evaluator, because the external timestamp shouldn't affect the prepare statement.